Signum.React/Scripts/Lines/EntityBase.tsx

```diff
@@ -264,6 +267,40 @@ export class EntityBaseController<P extends EntityBaseProps> extends LineBaseCon
     }).done();
   };
 
+  handlePasteClick = (event: React.SyntheticEvent<any>) => {
+
+    event.preventDefault();
+
+    navigator.clipboard.readText()
+      .then(text => {
+        const lites = parseLiteList(text);
+        if (lites.length == 0)
+          return;
+
+        const ti = this.props.type!.name == IsByAll ? getTypeInfo(lites[0].EntityType) : getTypeInfos(this.props.type!).singleOrNull(ti => ti.name == lites[0].EntityType);

```

Comment:
again better to filter the lites that are compatible. Maybe you pasting a list of apples and oranged in a orange field.

Reply:
done 👍